pkg/manifest/manifest_loader_test.go

```diff
@@ -22,15 +22,18 @@ import (
 	"fmt"
 	"github.com/dynatrace/dynatrace-configuration-as-code/internal/log"
 	monacoVersion "github.com/dynatrace/dynatrace-configuration-as-code/internal/version"
+	"github.com/dynatrace/dynatrace-configuration-as-code/pkg/client"
 	"github.com/dynatrace/dynatrace-configuration-as-code/pkg/version"
 	"github.com/google/go-cmp/cmp"
 	"github.com/google/go-cmp/cmp/cmpopts"
 	"github.com/spf13/afero"
+	"gopkg.in/yaml.v2"
 	"math"
 	"reflect"
 	"testing"
 
-	"gotest.tools/assert"
+	"github.com/stretchr/testify/assert"
+	assert2 "gotest.tools/assert"
```

Comment:
Two complex cases left to migration on `github.com/stretchr/testify/assert`

``` go
assert2.DeepEqual(t, test.expectedManifest, mani, cmp.AllowUnexported(EnvironmentDefinition{}))
```

and 
``` go
assert2.DeepEqual(t, got, tt.want, cmpopts.SortSlices(func(a, b ProjectDefinition) bool { return a.Name < b.Name }))
```

Reply:
Since we agreed that we do either all or nothing in a file, I checked how we can do this, and it's really trivial to replace the code.
Simply use assert.Equal: `assert.Equal(t, got, tt.want)` and `assert.Equal(t, test.expectedManifest, mani)`

Please fix the code only to use testify